tests/unit/deploy/test_export.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for exporting."""
```

Comment:
I don't think we need this test. We already add `engine.export` in one of the open PRs

Reply:
Which one? In this case, I'm not testing the export function but the ability to export all models to OpenVINO. All of them must work in OV. Should I move this test to another directory, then?

Reply:
The problem is that currently the video models cannot be exported, and some other models have incorrect graphs due to frozen ops, unrolled for-loops based on example tensor dimensions, and flattened conditionals. While ideally we would like to support OpenVINO conversion for all the models, it might require some extra effort that might be out of scope for this PR. That's why I feel we should drop this test. Also, personally I feel unit tests should not test for export coverage. It should just test the particular method. We can include a test for export coverage later in integration tests or nightly. Maybe other have a different opinion.

Reply:
I'm gonna move all OpenVINO export tests to a separate PR.

Reply:
#1561